homeassistant/core.py

```diff
@@ -1327,6 +1333,10 @@ def __repr__(self) -> str:
         return f"<_OneTimeListener {self.listener_job.target}>"
 
 
+# Empty list, used by EventBus._async_fire
+EMPTY_LIST: list[Any] = []
```

Comment:
I think this should better be annotated as `list[Never]`, but mypy does not like that